taskcluster/fenix_taskgraph/transforms/release_deps.py

```diff
@@ -0,0 +1,45 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+"""
+Add dependencies to release tasks.
+"""
+
+from taskgraph.transforms.base import TransformSequence
+
+PHASES = ["build", "promote", "push", "ship"]
+
+transforms = TransformSequence()
+
+
+@transforms.add
+def add_dependencies(config, tasks):
```

Comment:
I now that I look at this transform again, I don't know if we could upstream it, maybe each project would have different phases? I guess there might be a way or parametrize that. I can see myself copy-pasting this transform in the future...

Reply:
I totally agree! I haven't ported it to standalone taskgraph because this transform is mobile-specific for now. The only reason is: the attribute it uses (`release-type` instead of `build-type` in gecko). I'll look into a way to make it more standard so that the name of the attribute doesn't matter that much anymore.